core/src/main/java/com/novoda/noplayer/internal/exoplayer/mediasource/AudioTrackType.java

```diff
@@ -0,0 +1,22 @@
+package com.novoda.noplayer.internal.exoplayer.mediasource;
+
+public enum AudioTrackType {
+    MAIN(0),
+    AUDIO_DESCRIBED(1),
```

Comment:
I'm not sure if we should be calling this `AudioDescribed` whilst the most common usecase is audio description it's not enforced by the dash spec 

Annoyingly I can't find the official DASH spec website any more only this which isn't great

http://dashif.org/software/